docs/src/main/sphinx/release/release-363.md

```diff
@@ -0,0 +1,63 @@
+# Release 363 (XX Oct 2021)
+
+## General
+
+* Reduce query memory usage when spill occurs. ({issue}`9270`)
+* Reduce join memory usage. ({issue}`9327`)
+* Reduce query memory usage during join unspilling. ({issue}`9275`)
+* Fix potential data loss in query results when clients retry requests to the coordinator. ({issue}`9453`)
+* Fix decimal arithmetic to not produce negative zeros. Previously comparing two zero decimal
+  values might have returned `false` because of dual zero representation. ({issue}`8985`)
```

Comment:
decimal "negative zeros" is not something end user should be able to understand. From user perspective, such thing has never existed.

Reply:
How about:

> Fix incorrect result for comparisons between zero-valued decimals.